tests/foreman/ui/test_organization.py

```diff
@@ -365,13 +382,15 @@ def test_positive_remove_domain(self):
 
         :CaseLevel: Integration
         """
+
         with Session(self.browser) as session:
+            org_name = _org_name_module_scope(session)
```

Comment:
I made the modification and will post the result. It's more clear and allow org reuse on all other UI tests as well. ty.